cosmos-proxy/src/cosmos-proxy.js

```diff
@@ -25,10 +25,18 @@ var http = require('http'),
     httpProxy = require('http-proxy'),
     url = require('url'),
     idm = require('./idm.js'),
-    logger = require('./logger.js'),
-    conf = require('../conf/cosmos-proxy.json');
-    cache = require('./cache.js');
-    tidoopfs = require('./tidoopfs.js');
+    conf = require('../conf/cosmos-proxy.json'),
+    cache = require('./cache.js'),
+    tidoopfs = require('./tidoopfs.js'),
+    helper = require('./conf_helper');
+    logger = require('./logger.js');
```

Comment:
Finally, it seems there is no problem if the logger is required before checking the validity of the configuration, right?